tests/unittests/BackendTest.cpp

```diff
@@ -246,6 +247,28 @@ TEST_P(BackendTest, BundleSharedConstant) {
   EXPECT_TRUE(it2 != table2.end());
 }
 
+/// Test that the symbol category for a symbol is properly set.
+TEST_P(BackendTest, BundleSymbolCategory) {
+  Module mod;
+  PlaceholderBindings bindings;
+  Function *F = mod.createFunction("main");
+  auto *X = mod.createConstant(ElemKind::FloatTy, {3}, "X");
+  X->getHandle() = {1., 2., 3.};
+  auto *pow = F->createPow("Pow1", X, 2.0);
+  auto *save = F->createSave("save", pow);
+  bindings.allocate(save->getPlaceholder());
+
+  std::unique_ptr<Backend> backend(createBackend(GetParam()));
+  auto function = backend->compile(F);
+  auto table1 = function->getRuntimeBundle().getSymbolTable();
+  /// Make sure X is in both tables.
+  auto it = table1.find(X->getName());
+  EXPECT_EQ(it->second.symbolCategory, glow::runtime::SymbolCategory::Constant);
+  it = table1.find(save->getName());
+  EXPECT_EQ(it->second.symbolCategory,
+            glow::runtime::SymbolCategory::Placeholder);
```

Comment:
May be add some tests for activations and tensorviews as well? In particular, there should be tests for checking that tensorviews have the same category as their underlying tensors.